src/Helper/CkoLogger.php

```diff
@@ -0,0 +1,79 @@
+<?php
+
+namespace Checkoutcom\Helper;
+
+use Checkoutcom\Config\Config;
+use GuzzleHttp\Psr7\Request;
+use GuzzleHttp\Client;
+use RuntimeException;
+use Checkoutcom\Helper\Utilities;
+use Psr\Log\LoggerInterface;
+
+class CkoLogger {
```

Comment:
We can just use `monolog` to support different transports (in monolog language this is a `handler`). Transport being shopware6 logger and our logging service (cloudevent API). We should be able to compose them together.

Documentation:
https://github.com/Seldaek/monolog/blob/main/doc/02-handlers-formatters-processors.md#handlers

Example handler:
- https://github.com/Seldaek/monolog/blob/main/src/Monolog/Handler/LogglyHandler.php
- https://github.com/Seldaek/monolog/blob/main/src/Monolog/Handler/ErrorLogHandler.php

Looks like we only need to implement `write` method which is specific for a handler (transport).